WebApp/WebContent/dataset/dataset_list.xhtml

```diff
@@ -33,6 +33,14 @@
         $('#datasetListForm\\:autoQcLink').click();
         return false;
       }
+
+      function triggerUpdateList() {
+        $('#datasetListForm\\:updateList').click();
```

Comment:
It would be more logical to handle these events using the PrimeFaces remoteCommand - construct, rather than clicking on hidden links. https://www.primefaces.org/showcase/ui/ajax/remoteCommand.xhtml